.github/workflows/scale-test-100-gce.yaml

```diff
@@ -0,0 +1,222 @@
+          ref: 67af7e2664b1a5433d098031db0253d5ed4d2a73 # tinker/learnitall/100-node-scale-test
+          persist-credentials: false
+          sparse-checkout: clusterloader2
+          path: perf-tests
+
+      - name: Display version info of installed tools
+        run: |
+          echo "--- go ---"
+          go version
+          echo "--- cilium-cli ---"
+          cilium version --client
+          echo "--- kops ---"
+          ./kops version
+          echo "--- gcloud ---"
+          gcloud version
+
+      - name: Deploy cluster
+        id: deploy-cluster
+        uses: ./.github/actions/scale-tests/create-cluster
+        timeout-minutes: 30
+        with:
+          cluster_name: ${{ steps.vars.outputs.cluster_name }}
+          control_plane_size: n1-standard-8
+          control_plane_count: 1
+          node_size: e2-medium
+          node_count: 100
+          kops_state: ${{ secrets.GCP_PERF_KOPS_STATE_STORE }}
+          project_id: ${{ secrets.GCP_PERF_PROJECT_ID }}
+
+      - name: Create IG for resource heavy deployments
+        uses: ./.github/actions/scale-tests/create-ig
+        timeout-minutes: 30
+        with:
+          cluster_name: ${{ steps.vars.outputs.cluster_name }}
+          node_size: e2-standard-8
+          node_count: 1
+          ig_name: heapster
+          kops_state: ${{ secrets.GCP_PERF_KOPS_STATE_STORE }}
+
+      - name: Setup firewall rules
+        uses: ./.github/actions/scale-tests/setup-firewall
+        with:
+          cluster_name: ${{ steps.vars.outputs.cluster_name }}
+
+      - name: Install Cilium
+        run: |
+          cilium install ${{ steps.vars.outputs.cilium_install_defaults }}
+
+      - name: Wait for cluster to be ready
+        uses: ./.github/actions/scale-tests/validate-cluster
+        timeout-minutes: 45
+        with:
+          cluster_name: ${{ steps.vars.outputs.cluster_name }}
+          kops_state: ${{ secrets.GCP_PERF_KOPS_STATE_STORE }}
+
+      - name: Wait for Cilium status to be ready
+        run: |
+          cilium status --wait
+
+      - name: Run CL2
+        id: run-cl2
+        working-directory: ./perf-tests/clusterloader2
+        run: |
+          mkdir ./report
+          export CL2_PROMETHEUS_PVC_ENABLED=false
+          export CL2_ENABLE_PVS=false
+          export CL2_ENABLE_NETWORKPOLICIES=true
+          export CL2_ALLOWED_SLOW_API_CALLS=1
+
+          go run ./cmd/clusterloader.go \
+            -v=4 \
+            --testconfig=./testing/load/config.yaml \
+            --provider=gce \
+            --enable-prometheus-server \
+            --tear-down-prometheus-server=false \
+            --nodes=100 \
+            --report-dir=./report \
+            --experimental-prometheus-snapshot-to-report-dir=true \
+            --kubeconfig=$HOME/.kube/config \
+            --testoverrides=./testing/overrides/load_throughput.yaml \
+            --testoverrides=./testing/experiments/use_simple_latency_query.yaml \
```

Comment:
I must have missed it when adding tee. Test https://github.com/cilium/cilium/actions/runs/6896887510/job/18763936013 failed with error:
```
measurement call SchedulingThroughput - SchedulingThroughput error: scheduler throughput: actual throughput 21.000000 lower than threshold 100.000000
```
This is fine, as 100 nodes cluster is expected to have ~20 throughput, but the test itself didn't fail, because of tee. 
We should set `set -o pipefail` before and also turn off scheduling throughput for this test with `CL2_SCHEDULER_THROUGHPUT_THRESHOLD=0` (this is what k8s OSS test does for 100 nodes test [here](https://github.com/kubernetes/test-infra/blob/a6a331d32674564c9b70da46127c77279c3081e5/config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml#L210) )

Reply:
I set the `shell: bash` parameter, which will set pipefail: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell